django-rgd/rgd/utility.py

```diff
@@ -214,7 +214,9 @@ def patch_internal_presign(f: FieldFile):
 
 
 @contextmanager
-def output_path_helper(filename: str, output: FieldFile):
+def output_path_helper(filename: str, output: ChecksumFile, final_name: str = None):
```

Comment:
```suggestion
def output_path_helper(filename: str, output: ChecksumFile, final_name: Optional[str] = None):
```

With `from typing import Optional` added to the top.